tasks/db_deploy/orca_sql.py

```diff
@@ -596,7 +596,7 @@ def add_multipart_chunksize_sql() -> TextClause:
         """
         ALTER TABLE recovery_file
         ADD COLUMN IF NOT EXISTS multipart_chunksize_mb integer NULL;
-        COMMENT ON COLUMN recovery_file.multipart_chunksize
+        COMMENT ON COLUMN recovery_file.multipart_chunksize_mb
             IS 'Overrides orca_default_multipart_chunksize in TF.';
```

Comment:
Is this orca_default_multipart_chunksize_mb or default_multipart_chunksize_mb in terraform?

Reply:
Whoops, hadn't saved that properly. Fixed.